test/tf/gen_tf_pb.py

```diff
@@ -142,19 +144,21 @@ def batchnormv3_test(g1):
     with g1.as_default():
         g1_input = tf.compat.v1.placeholder(tf.float32,
                                             shape=(1, 16, 16, 32),
-                                            name='0')
-        g1_scale = tf.constant(1.0, dtype=tf.float32, shape=[32], name='1')
-        g1_offset = tf.compat.v1.placeholder(tf.float32, shape=(32), name='2')
-        g1_mean = tf.compat.v1.placeholder(tf.float32, shape=(32), name='3')
+                                            name='x')
+        g1_scale = tf.constant(1.0, dtype=tf.float32, shape=[32], name='scale')
+        g1_offset = tf.compat.v1.placeholder(tf.float32,
+                                             shape=(32),
+                                             name='bias')
+        g1_mean = tf.compat.v1.placeholder(tf.float32, shape=(32), name='mean')
         g1_variance = tf.compat.v1.placeholder(tf.float32,
                                                shape=(32),
-                                               name='4')
+                                               name='variance')
         tf.raw_ops.FusedBatchNormV3(x=g1_input,
                                     scale=g1_scale,
                                     offset=g1_offset,
                                     mean=g1_mean,
                                     variance=g1_variance,
-                                    epsilon=0.00001,
+                                    epsilon=1e-6,
```

Comment:
We're more precice for this test but less on the other test for batchnorm_test() above, why is that?

Reply:
Testing that the epsilon value is being passed over correctly.

Reply:
Oh that makes sense.